package.json

```
@@ -0,0 +1,15 @@
+{
+  "version": "0.0.1",
+  "private": true,
+  "description": "A library for 0Auth that authenticates using locally stored personal information instead of OAuth",
+  "repository": "https://github.com/0-Auth/0Auth.git",
```

Because we suggest multiple libraries as the package, this way is better I think.
```suggestion
  "description": "A libraries for 0Auth that authenticates using locally stored personal information instead of OAuth",
```

---

Checking the package-json of material-ui, it seems better to delete the description and repository from the top-level package.json.